### ISSUE TITLE
Configuration property 'trimHeaderLine' doesn't take any effect.

### ISSUE THREAD

**Issue by xeagle2** [mojohaus/license-maven-plugin#128]
Hi there,

Used version of the license-maven-plugin is 1.16.

When the 'trimHeaderLine' is set, it doesn't take any effect because every xxxTransformer implementation inherits AbstractFileHeaderTransformer where 

```
public void setTrimHeaderLine( boolean trimLine )
{
    this.emptyLineAfterHeader = trimLine;
}
```
is assigned to this.**emptyLineAfterHeader** **instead of** this.**trimHeaderLine**.

It looks like an issue which would be useful to be resolved.

Thank you,

**Comment by cjohnstoniv** [comment]
Has this been resolved? If so, can this be patched into a 1.16.1 release anytime soon?

**Comment by ppalaga** [comment]
The fix seems to have been merged. Feel free to reopen if that's not the case.
